### PR TITLE
Implemente Base behavior

### PR DESCRIPTION
**Pull request by thet**
There are some TODO items, like reviewing if the decision to leave out some deprecated attributes‌ (according to the comments from recensio.contenttypes) or grouping fields in special fieldsets.

IMO these can be reviewed and resolved later, so I think this PR is ready to be reviewed and merged.

### CODE REVIEW

File: src/recensio/plone/behaviors/base.py [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+    )
+    directives.omitted("recensioID")
+
+    generatedPdf = namedfile.NamedBlobFile(
+        title=_("Generated Pdf"),
+        required=False,
+    )
+    directives.omitted("generatedPdf")
+    directives.no_omit(IEditForm, "generatedPdf")
+    directives.no_omit(IAddForm, "generatedPdf")
```
Comment (reinhardt) [inline src/recensio/plone/behaviors/base.py:85]: In the old code this field is completely hidden by setting the widget condition to `False`.

https://github.com/syslabcom/recensio.contenttypes/blob/master/recensio/contenttypes/content/schemata.py#L143

File: src/recensio/plone/behaviors/base.py [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+
+    ppn = schema.TextLine(
+        title=_("PPN"),
+        description=_(
+            "description_bv_number", default="Filled in by the editorial staff"
+        ),
+        required=False,
+    )
+
+    canonical_uri = schema.URI(
```
Comment (reinhardt) [inline src/recensio/plone/behaviors/base.py:117]: This is an `atapi.StringField` in the old code. Could there be issues when changing to `schema.URI`? If I'm not mistaken the validators differ (old recensio does not require a scheme).

File: src/recensio/plone/behaviors/base.py (hunk 1 above)
Comment (ale-rt) [inline src/recensio/plone/behaviors/base.py:85]: @reinhardt, so this:
```suggestion
```
Is fixing your conncerns, right?

File: src/recensio/plone/behaviors/base.py (hunk 2 above)
Comment (ale-rt) [inline src/recensio/plone/behaviors/base.py:117]:
```suggestion
    canonical_uri = schema.TextLine(
```

File: src/recensio/plone/behaviors/base.py (hunk 1 above)
Comment (reinhardt) [reply src/recensio/plone/behaviors/base.py:85]: Yes, I think so.

**Review by reinhardt — APPROVED** [review]
FWIW the presentation types also need the base behavior, but we can take care of that later.

**Pull request merged**